frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-item.tsx

```diff
@@ -1,15 +1,12 @@
 import * as React from 'react';
 import * as _ from 'lodash';
-import { LoadingInline, pluralize } from '@console/internal/components/utils';
 import { RedExclamationCircleIcon } from '@console/shared';
-
-const pluralizeWithNotation = (i: number, suffix: string, singular: string) =>
-  `${i}${suffix} ${i === 1 ? singular : `${singular}s`}`;
+import { LoadingInline, pluralize, humanizeDecimalBytes } from '@console/internal/components/utils';
 
 const formatCount = (count: number) => {
-  if (count < 1000) return pluralizeWithNotation(count, '', 'Object');
-  if (count >= 1000 && count < 1000000) return pluralizeWithNotation(count / 1000, 'K', 'Object');
-  return pluralizeWithNotation(count / 1000000, 'M', 'Object');
+  const hCount = humanizeDecimalBytes(count);
+  // suppressing the end 'B' in units
```

Comment:
This is a side-effect of using the wrong humanize function, which we should avoid altogether :)

Reply:
Thanks for the pointers @spadgett , corrected the humanize function (used `humanizeNumber`)

![image](https://user-images.githubusercontent.com/895234/62834671-5f6dcf80-bc6d-11e9-9095-c2dd9ee023ad.png)

![image](https://user-images.githubusercontent.com/895234/62834757-7bbe3c00-bc6e-11e9-9f0c-481c6bc8a620.png)